toolkit/primitives/domain/src/lib.rs

```diff
@@ -418,6 +418,14 @@ impl From<ecdsa::Public> for SidechainPublicKey {
 	}
 }
 
+#[derive(Clone, Encode, Decode, TypeInfo, PartialEq, Eq, Hash)]
+#[byte_string(debug, hex_serialize, hex_deserialize, decode_hex)]
+pub struct TransactionCbor(pub Vec<u8>);
+
+#[derive(Clone, Encode, Decode, TypeInfo, PartialEq, Eq, Hash)]
+#[byte_string(debug, hex_serialize, hex_deserialize, decode_hex)]
+pub struct VKeyWitnessCbor(pub Vec<u8>);
```

Comment:
Please add doc string. Please provide a hint what type is serialized and if any header is there. I think referring to a type from some core Cardano library would be proper here.